app/adapters/tumblr-post-photo.js

```diff
@@ -0,0 +1,5 @@
+import PostAdapter from './tumblr-post';
```

Comment:
This file should live in `addon/adapters/tumblr-post-photo`. Files in `app/` should be rexports only, no extra code.